packages/@coorpacademy-components/src/molecule/video-player/jwplayer.js

```diff
@@ -9,10 +9,18 @@ import style from './jwplayer.css';
 class JWPlayer extends React.Component {
   constructor(props, context) {
     super(props, context);
+    this.state = {
+      fileUrl: ''
+    };
     this.handlePlay = this.handlePlay.bind(this);
     this.handleResume = this.handleResume.bind(this);
     this.handlePause = this.handlePause.bind(this);
     this.handleEnded = this.handleEnded.bind(this);
+    this.handleError = this.handleError.bind(this);
+  }
+
+  componentDidMount() {
+    this.setFileUrl();
```

Comment:
this thing should be done in didUpdate or didReceiveProps to ensure that the url can change even if the component doesn't unmount